pulsar-broker-common/src/main/java/org/apache/pulsar/broker/resources/NamespaceResources.java

```diff
@@ -49,16 +49,18 @@ public class NamespaceResources extends BaseResources<Policies> {
     private final IsolationPolicyResources isolationPolicies;
     private final PartitionedTopicResources partitionedTopicResources;
     private final MetadataStore configurationStore;
+    private final MetadataStore localStore;
 
     public static final String POLICIES_READONLY_FLAG_PATH = "/admin/flags/policies-readonly";
     private static final String NAMESPACE_BASE_PATH = "/namespace";
     private static final String BUNDLE_DATA_BASE_PATH = "/loadbalance/bundle-data";
 
-    public NamespaceResources(MetadataStore configurationStore, int operationTimeoutSec) {
+    public NamespaceResources(MetadataStore localStore, MetadataStore configurationStore, int operationTimeoutSec) {
```

Comment:
How about creating a separate class naming `BundleResources` to handle the bundle-date creating and deleting.  The reason is that:

1. `NamespaceResources` will be null if `configurationMetadataStore != null` is false, so there is a chance to throw NPE when  call `NamespaceResources.deleteBundleDataAsync(..)` or `NamespaceResources.deleteBundleDataTenantAsync(...)`:

https://github.com/apache/pulsar/blob/d099ac4fa2f217b9c5f0a5e660c83048e829c5d7/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/resources/PulsarResources.java#L60-L64

2. `NamespaceResources#BUNDLE_DATA_BASE_PATH` and `ModularLoadManagerImpl#BUNDLE_DATA_PATH` are duplicated.  `BundleResources` could unify them. WDYT

Reply:
1. It seems that we can't allow the `configurationMetadataStore` to be null, or there will be serious issues.
https://github.com/apache/pulsar/blob/d099ac4fa2f217b9c5f0a5e660c83048e829c5d7/pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java#L752-L762

2. Currently, bundle-data are created/updated by `LoadManager`, and deleted by `LoadManager` and `NamespaceResources`. It is beneficial to introduce a `BundleResources` to unify the opeartions to bundle-data. But i think that it should be introduced as a code optimization instead of bug fix, which need to be discussed fully.

Reply:
OK. As a bug fix, it change could fix the problem and LGTM. Maybe we could refactor in another PR